### ISSUE TITLE
Investigate optimal EC2 node type for generators, and identify maximum number of VUs per node 

### ISSUE THREAD

**Issue by westrachel** [team-aglr/edemame#14]
(1) In utilities/eksctl.js, the createLoadGenGrp() function creates a node group with max 100 nodes and with node types set to "t3.small". These are arbitrary selections. I don't know that we'd ever need more than 100 nodes, so that choice may be fine, but we might want to select a different type of node instead of t3.small. 

(2) As part of 1, what type of node is optimal may depend on whether or not we can configure 1 pod per 1 node. Right now we are not constraining 1 pod to 1 node even though we are creating a load generator group.

(3) when we create a cluster with eksctl, it creates 2 m5.large nodes, is there a more optimal node type for our different server components?

(4) also, we don't know yet how many VUs we want per pod/node. Once we know that we need to update NUM_VUS_PER_POD in constants/constants.js

**Comment by lukeoguro** [comment]
I think we need to:
1. Investigate how many VUs we want per EC2 node (and which type of EC2 node).
2. Make sure that the k6 CRD (custom resource definition) is specified so that there is 1 pod per node.
3. Update `createLoadGenGrp` so that the correct node type is chosen.

**Comment by elguaposalsero** [comment]
This might be useful (prometheus)

https://www.youtube.com/watch?v=wa3e6Ery9bg

**Comment by lukeoguro** [comment]
See [Running large tests](https://k6.io/docs/testing-guides/running-large-tests/)

OS should be "fine-tuned"

```
sysctl -w net.ipv4.ip_local_port_range="1024 65535"
sysctl -w net.ipv4.tcp_tw_reuse=1
sysctl -w net.ipv4.tcp_timestamps=1
ulimit -n 250000
```

**Comment by lukeoguro** [comment]
Resolved in #107
